S11-modules/distribution.t

```diff
@@ -0,0 +1,43 @@
+use v6;
+constant $repo-path = $?FILE.IO.parent(2).child('packages/distribution').absolute;
```

Comment:
this isn't even an existing path -- nor should any additional libs be needed for such a test

Reply:
OK